src/elements/Typography/Typography.module.css

```diff
@@ -0,0 +1,50 @@
+.title {
+  font-size: 3rem;
```

Comment:
colors and font sizes here should correspond to what we have in the theme files

Reply:
Ya - sorry this isn't final - just starting it - please continue w/ feedback